test/classes/casts/generated-cast-tests/.gitignore

```diff
@@ -0,0 +1,10 @@
+argumentCoerce_Const
+argumentCoerce_ConstIn
+argumentCoerce_ConstRef
+argumentCoerce_In
+argumentCoerce_Inout
+argumentCoerce_Out
+argumentCoerce_Ref
+coerceAssign
+coerceInitAssign
+explicitCasts
```

Comment:
Are these the names of the binaries generated by the tests?  If so, I'm trying to think why we don't need these more generally for every test we file.  My guess is that it's because the test system typically cleans them up after running them.  Is this directory different in some way?  (or maybe it was before the `./PRETEST` change?

Reply:
These are the directory names generated by `generate-tests`. I opted to not delete the generated tests after running them (with a `POSTTEST` or some other mechanism). I think it is valuable to have the source code to inspect if one of these tests should fail, instead of wondering "well this mysterious test fails and I can't find it". And this was an easy way to exclude the generated directories from source control

Reply:
Great, thanks for clarifying (and covering for my laziness in not looking deeper)—definitely reassuring!